### ISSUE TITLE
Install pytest_xprocess module requires non-installed _internal module

### ISSUE THREAD

**Issue by mgorny** [pytest-dev/pytest-xprocess#80]
**Describe the bug**
The package installs `pytest_xprocess` module by default:

```
py_modules = 
	pytest_xprocess
	xprocess
```

However, this module requires `_internal` that isn't (and with that name, mustn't be) installed:

```python
from _internal import getrootdir
```

Therefore, after installing it pytest keeps erroring out due to missing module, e.g.:

```pytb
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.10/pytest", line 33, in <module>
    sys.exit(load_entry_point('pytest==6.2.4', 'console_scripts', 'pytest')())
  File "/usr/lib/python3.10/site-packages/_pytest/config/__init__.py", line 185, in console_main
    code = main()
  File "/usr/lib/python3.10/site-packages/_pytest/config/__init__.py", line 143, in main
    config = _prepareconfig(args, plugins)
  File "/usr/lib/python3.10/site-packages/_pytest/config/__init__.py", line 318, in _prepareconfig
    config = pluginmanager.hook.pytest_cmdline_parse(
  File "/usr/lib/python3.10/site-packages/pluggy/hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/usr/lib/python3.10/site-packages/pluggy/manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/usr/lib/python3.10/site-packages/pluggy/manager.py", line 84, in <lambda>
    self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(
  File "/usr/lib/python3.10/site-packages/pluggy/callers.py", line 203, in _multicall
    gen.send(outcome)
  File "/usr/lib/python3.10/site-packages/_pytest/helpconfig.py", line 100, in pytest_cmdline_parse
    config: Config = outcome.get_result()
  File "/usr/lib/python3.10/site-packages/pluggy/callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/usr/lib/python3.10/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/usr/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1003, in pytest_cmdline_parse
    self.parse(args)
  File "/usr/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1283, in parse
    self._preparse(args, addopts=addopts)
  File "/usr/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1172, in _preparse
    self.pluginmanager.load_setuptools_entrypoints("pytest11")
  File "/usr/lib/python3.10/site-packages/pluggy/manager.py", line 299, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 162, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "/usr/lib/python3.10/site-packages/_pytest/assertion/rewrite.py", line 170, in exec_module
    exec(co, module.__dict__)
  File "/usr/lib/python3.10/site-packages/pytest_xprocess.py", line 4, in <module>
    from _internal import getrootdir
ModuleNotFoundError: No module named '_internal'
```

**To Reproduce**
1. Install the package.
2. Attempt to run `pytest` anywhere.

**Expected behavior**
Working pytest.

**Environment (please complete the following information):**
 - OS: Gentoo Linux
 - Python Version 3.10.0b4
 - pytest-xprocess version 0.18.0

**Comment by RonnyPfannschmidt** [comment]
fixed by #78 and released in #79

**Comment by mgorny** [comment]
Thanks, I can confirm that the new release works for me.
